models/league.js

```diff
@@ -3,12 +3,20 @@ const yaml = require('js-yaml');
 const logger = require('../logger.js').child({ module: 'league' });
 const { Coach } = require('./coaches.js');
 const { Round } = require("./round.js");
+const { processConfigValue } = require("./utils/config-reader.js");
 
 /** A Blood Bowl league. */
 class League {
     constructor(data, leagueFile) {
         this.name = data.name;
-        this.ownerId = data.ownerId || data.ownerID;
+
+        //OwnerId
+        this.ownerIdRaw = data.ownerId || data.ownerID;
+        const ownerIdOrError = processConfigValue(this.ownerIdRaw);
+        logger.debug(`ownerIdOrError = ${ownerIdOrError}`);
+        ownerIdOrError.onLeft((v) => this.ownerId = v);
```

Comment:
If `Either` is reference to the `Either` monad I googled just now, ours is backwards: `Right` contains a result and `Left` contains an error. https://monet.github.io/monet.js/#either

Reply:
Yeah I was basing it on the type syntax:

Either<String, Error> is String on the Left, Error on the Right

Either<Error, String> would be the other way.

Atm it doesn't matter because I haven't implemented any Applicative or Monadic laws where which type gets mapped matters